eng/testing/tests.targets

```diff
@@ -162,47 +178,44 @@
        This target is executed once build is done for a test lib (after CopyFilesToOutputDirectory target) -->
   <UsingTask TaskName="AppleAppBuilderTask" 
              AssemblyFile="$(ArtifactsObjDir)mono\AppleAppBuilder\$(TargetArchitecture)\$(Configuration)\AppleAppBuilder.dll" />
-  <Target Condition="'$(TargetOS)' == 'iOS'" Name="BundleTestAppleApp" AfterTargets="CopyFilesToOutputDirectory">
-    <PropertyGroup>
-      <RuntimePackDir>$(ArtifactsDir)bin\lib-runtime-packs\runtimes\ios-$(TargetArchitecture)</RuntimePackDir>
-      <BundleDir>$(OutDir)\Bundle</BundleDir>
-      <AppleTestRunner>$(RepoRoot)\src\mono\msbuild\AppleTestRunner\bin</AppleTestRunner>
-    </PropertyGroup>
-    <!-- We need to copy additional stuff into $(OutDir)\Bundle
-         1) The whole BCL
-         2) Test Runner (with xharness client-side lib)
-      -->
+  <UsingTask TaskName="MonoAotCompiler"
+             AssemblyFile="$(ArtifactsObjDir)mono\MonoAotCompiler\$(TargetArchitecture)\$(Configuration)\MonoAotCompiler.dll" />
+
+  <Import Project="$(MonoProjectRoot)msbuild\AotCompilerTask\MonoAotCompiler.props" />
+  <Target Name="AOTCompileAssemblies" Condition="'$(RunAOTCompilation)' != ''" DependsOnTargets="CopyAssembliesToBundleDir">
     <ItemGroup>
-      <TestBinaries Include="$(OutDir)\*.*"/>
-      <AppleTestRunnerBinaries Include="$(AppleTestRunner)\*.*" />
-      <BclBinaries Include="$(RuntimePackDir)\lib\$(NetCoreAppCurrent)\*.*" Exclude="$(RuntimePackDir)\lib\$(NetCoreAppCurrent)\System.Runtime.WindowsRuntime.dll" />
-      <BclBinaries Include="$(RuntimePackDir)\native\*.*" Exclude="$(RuntimePackDir)\native\libmono.dylib" /> <!-- we use static libmono.a -->
-      
-      <!-- remove PDBs to save some space until we integrate ILLink -->
-      <BclBinaries Remove="$(RuntimePackDir)\lib\$(NetCoreAppCurrent)\*.pdb" />
+      <AotInputAssemblies Include="$(BundleDir)\*.dll">
+        <AotArguments>@(MonoAotCompilerDefaultAotArguments, ';')</AotArguments>
+        <ProcessArguments>@(MonoAotCompilerDefaultProcessArguments, ';')</ProcessArguments>
+      </AotInputAssemblies>
     </ItemGroup>
-    <Error Condition="!Exists('$(AppleTestRunner)')" Text="AppleTestRunner=$(AppleTestRunner) doesn't exist" />
-    <Error Condition="!Exists('$(RuntimePackDir)')" Text="RuntimePackDir=$(RuntimePackDir) doesn't exist" />
-    <RemoveDir Directories="$(BundleDir)" />
-    <Copy SourceFiles="@(TestBinaries)" DestinationFolder="$(BundleDir)" SkipUnchangedFiles="true"/>
-    <Copy SourceFiles="@(AppleTestRunnerBinaries)" DestinationFolder="$(BundleDir)\%(RecursiveDir)" SkipUnchangedFiles="true"/>
-    <Copy SourceFiles="@(BclBinaries)" DestinationFolder="$(BundleDir)\%(RecursiveDir)" SkipUnchangedFiles="true"/>
+    <MonoAotCompiler
+        AotCompilerBinary="$(RuntimePackDir)\native\cross\mono-aot-cross"
```

Comment:
You should be able to use `<NETCoreAppRuntimePackNativePath>` as the root part of the path.